### DISCUSSION TITLE
Framerate limit setting?

### DISCUSSION THREAD

**Discussion by slayde1970** [Tape-Worm/Gorgon#39] [Q&A] [answered]
It appears the FPS is capped to 60 by default. Is there a setting to allow a higher frame rate?

**Comment by Tape-Worm** [comment, marked as answer]
Like it says in the documentation for GorgonSwapChain.Present:
>If the _interval_ parameter is greater than 0, then this method will synchronize to the vertical blank count specified by interval.  Passing 0 will display the contents of the back buffer as soon as possible.

Which basically means if you omit or pass 0 to the method parameter, then it will run uncapped.
